### DISCUSSION TITLE
Password show-password-on="mousedown" works only in desktop devices

### DISCUSSION THREAD

**Discussion by yariksav** [tusen-ai/naive-ui#1287] [General]
Password Inputs props "show-password-on"="mousedown" work only in desktop devices, mobile devices have "touchstart", "touchend" events and don't handle mouse events...

  el.addEventListener("touchstart", handleStart, false);
  el.addEventListener("touchend", handleEnd, false);

https://github.com/TuSimple/naive-ui/blob/70399bcd4b439ac2d977d8eb105cbb71b175cd40/src/input/src/Input.tsx#L1009

**Comment by 07akioni** [comment]
Are you interested in creating a PR?
